<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>librepcb::application::InitializeWorkspaceWizard_ChooseSettings</class>
 <widget class="QWizardPage" name="librepcb::application::InitializeWorkspaceWizard_ChooseSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose settings</string>
  </property>
  <property name="title">
   <string>Choose workspace settings</string>
  </property>
  <property name="subTitle">
   <string>Set the most important workspace settings.</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>User Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="edtUserName"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Language:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbxAppLanguage">
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>This name will be used as author when creating new projects or libraries.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Length Unit:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbxLengthUnit">
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
